### ISSUE TITLE
Network Receiver not working in other dimensions

### ISSUE THREAD

**Issue by Derel808** [raoulvdberge/refinedstorage#2194]
#### Issue description:
Trying to connect a Network Receiver (In Nether) to Network Transmitter (In Overworld)

#### What happens:
Network Receiver is not powered, so I can't connect any other RS blocks/ use Wireless Crafting Grid while near it in the Nether.

#### What you expected to happen:
Network Receiver to be powered from Network Transmitter which is connected to Controller which is powered, along with the Network Card in the Network Transmitter. The Network Card also has the coordinates of the Network Receiver, and it says "Dimension -1".

#### Steps to reproduce:
1. Place Network Receiver in Nether.
2. Right-click Network Receiver with Network Card.
3. Place Network Card in Network Transmitter in Overworld.
4. Go back to the Nether and Network Receiver should be powered.

#### Additional Information
I play Stone Block 1. I tried looking up solutions on YouTube, all browsing sites, and guides; couldn't find any guides except when the interdimensional upgrade was still in use. From what I understand, the Network Transmitter has been updated so that it is cross dimensional by default. And I've chunk loaded both areas in each dimension.
I've also looked through the threads here in GitHub and I couldn't find anything related.
Sorry if there is a simple solution to this, but I can't figure it out. 
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.12.2
- Forge: 14.23.5.2808
- Refined Storage: 1.6.12

Does this issue occur on a server? [yes/no]
No.
#### If a (crash)log is relevant for this issue, link it here:
No crashes.
[pastebin/gist/etc link here]

**Comment by Aang23** [comment]
Be sure both of them are chunkloaded.

**Comment by Derel808** [comment]
I did.
UPDATE: I also tried setting a Network Receiver in the Aroma Mining Dimension, also didn't work. But my Controller in the Overworld is showing that one of my Receivers in either the Nether / Mining Dimension is using power. I went to go check and neither is powered.

**Comment by Derel808** [comment]
UPDATE 2:
It works now. I just had to wait 30 in- game days or so until it registered. Not exactly sure what the issue was or whether it was performance lag. I have 16 gigs of Ram and allocated 8 gigs to Minecraft. Guess you just have to wait if you don't have a lot of RAM?
Thanks for the feedback 👍

**Comment by raoulvdberge** [comment]
The problem most likely lies with the chunkloading mod.

**Comment by psycomaffo** [comment]
I think I solved it simply ... I had the same problem, both in overworld and nether the chunks were loaded, placed the receiver in the nether with a crafting grid, linked the card, returned to overworld and placed the card in the transmitter and the controller did not detect the receiver, in fact the crafting grid in the nether was not powered.
I went again in overworld, removed the card from the transmitter and immediately replaced it and the controller recognized the receiver.
To be sure I did the same operation 5 times and all 5 times it worked the same.
I know it's a necro post, but as many mods are still on 1.12 I thought it might help
if I wasn't supposed to post, excuse me
And sry for my bad english
